### ISSUE TITLE
Spotlight not finding directories with spaces in their names

### ISSUE THREAD

**Issue by andy-js** [Netatalk/netatalk#61]
Searching for a directory name that contains spaces (in the root of a share) returns no results.  The same thing in a subdirectory works fine (provided you have #15)

**Comment by andylemin** [comment]
@rdmark is this resolved now?

**Comment by rdmark** [comment]
yes, with the recent spotlight overhaul this works with the localsearch backend
